tests/unit/__snapshots__/AppHeader.spec.js.snap

```diff
@@ -56,7 +56,7 @@ exports[`AppHeader.vue should render correctly 1`] = `
           <img
             alt="Built on OpenBMC"
             class="header-logo"
-            src="@/assets/images/logo-header.svg"
+            src="http://localhost/img/logo-header.svg"
```

Comment:
This is correct..?

Reply:
Do we need to change the test?

Reply:
This does look a little off but test is automatically updating to this. Will give it one more day to figure out if I can do something about this.

Reply:
I just checked without this change and it worked fine, Would undo of this snap file cause any issue?